### PR TITLE
Add all the files

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
finishing all files and pushing them for this project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <link rel="stylesheet" href="./style.css" />
+    <title>mobile skeleton</title>
+  </head>
+  <body>
+    <div class="header-container">
+      <h3>welcome</h3>
+      <div class="dropdown-menu">
+        <img src="./imges/dropdown.svg" alt="dropdown" />
+      </div>
+    </div>
```
Comment (mricanho) [inline index.html]: - [x] The container of the section should be a header tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>mobile skeleton</title>
+  </head>
+  <body>
+    <div class="header-container">
+      <h3>welcome</h3>
+      <div class="dropdown-menu">
+        <img src="./imges/dropdown.svg" alt="dropdown" />
+      </div>
```
Comment (mricanho) [inline index.html]: - [x] The hamburger button should be a `nav` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        Hey there. I’m yahya <br />
+        <span> I’m a software developer </span>
+      </h1>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <img src="./imges/social media icons.svg" alt="social-media-icons" />
+    </div>
```
Comment (mricanho) [inline index.html]: - [x] The container of the section should be a `section` tag, please fix it

File: index.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      <h1>
+        Hey there. I’m yahya <br />
+        <span> I’m a software developer </span>
+      </h1>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <img src="./imges/social media icons.svg" alt="social-media-icons" />
```
Comment (mricanho) [inline index.html]: - [x] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Right now you are using an image, please, fix it.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
+body {
+  max-width: 500px;
+  margin: auto;
+  background-image: url("./imges/header-illsutration-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
```
Comment (mricanho) [inline style.css]: - [x] Right now, the elements of the background are following the page when scrolling, erase this to fix that issue. [LIVE DEMO](https://rawcdn.githack.com/yahyaelganyni1/mobile-version-skeleton/feature/setup-project/index.html)

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @yahyaelganyni1, 

This is a very good first step to finish your project, you code most of this milestone, good use of CSS, strong knowledge of flexbox, and you are following the gitflow correctly.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [x] Remember that you need a professional README file, please add one.
- [x] Your PR title and description must be professionals, take a step back and fix them
- [x] From now on, you need to do more commit messages, just one commit it's not a good practice.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

a lot of work
It seems like a lot of work, but once you finish your project will be awesome.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (henatan99) [reply index.html]: This issue is addressed. Nice work! :clap:

File: index.html (hunk 2 above)
Comment (henatan99) [reply index.html]: This issue is addressed. Nice work! :clap:

File: index.html (hunk 3 above)
Comment (henatan99) [reply index.html]: This issue is addressed. Nice work! :clap:

File: index.html (hunk 4 above)
Comment (henatan99) [reply index.html]: This issue is addressed. Nice work! :clap:

File: style.css (hunk 5 above)
Comment (henatan99) [reply style.css]: - [ ] This issue is not resolved. See the attached gif of how the background behaves on your page and compare it with the demo page provided by the previous TSE. You can apply the same styling for the background image as on the demo page. 
chrome-capture

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi Yahya, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 6]
```diff
@@ -0,0 +1,87 @@
+body {
+  max-width: 500px;
+  margin: auto;
+  background-image: url("./imges/header-illsutration-mobile.svg");
```
Comment (henatan99) [inline style.css:4]: As the previous TSE stated, the background image should not move along the scroll bar. You can use the styling for the background image on the demo page provided by the previous TSE. [Demo](https://rawcdn.githack.com/yahyaelganyni1/mobile-version-skeleton/feature/setup-project/index.html)

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi Yahya, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (yahyaelganyni1) [reply style.css]: hello @henatan99 I just did what the first reviewer asked me to do I don't understand what should I do I mean in the mobile view it's working from the first time
I need more directing

**Review by descholar-ceo — APPROVED** [review]
Hi @yahyaelganyni1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**